src/messages/MPGStats.h

```diff
@@ -44,6 +44,31 @@ class MPGStats final : public PaxosServiceMessage {
   void print(std::ostream& out) const override {
     out << "pg_stats(" << pg_stat.size() << " pgs seq " << osd_stat.seq << " v " << version << ")";
   }
+  void dump_stats(ceph::Formatter *f, bool with_net = true) const {
+    f->open_object_section("stats");
+
+    f->open_array_section("pg_stat");
+    for(auto& s : pg_stat) {
+      f->open_object_section("pg_stat");
+      s.first.dump(f);
+      s.second.dump(f);
+      f->close_section();
+    }
+    f->close_section();
+
+    f->dump_object("osd_stat", osd_stat);
+
+    f->open_array_section("pool_stat");
+    for(auto& s : pool_stat) {
```

Comment:
same here.
And if not - we should probably change the problematic dump function.